mkdocs.yml

```diff
@@ -3,12 +3,20 @@ site_dir: 'rendered-docs'
 # Not having this (or viewing the site locally, or from any place other than the
 # site_url) leads to version switching failing to preserve the current path.
 site_url: 'https://jj-vcs.github.io/jj/'
+repo_url: https://github.com/jj-vcs/jj
+repo_name: jj-vcs/jj
+edit_uri: edit/main/docs/
 theme:
   name: 'material'
   language: 'en'
   favicon: images/favicon-96x96.png
   features:
     # - navigation.top
+    - content.action.edit
+
+  icon:
+    repo: fontawesome/brands/github
```

Comment:
Changed from the default:

![image](https://github.com/user-attachments/assets/31bd6ca5-3b73-4f69-90eb-1992656b838c)